srcpkgs/antimicro-qt4/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'antimicro-qt4'
+pkgname=antimicro-qt4
+version=2.23
+revision=1
+wrksrc="antimicro-${version}"
+build_style=cmake
+configure_args="-DUSE_SDL_2=ON -DWITH_UINPUT=ON -DWITH_XTEST=ON -DAPPDATA=ON"
+hostmakedepends="pkg-config itstool"
+makedepends="libXtst-devel SDL2-devel qt-devel"
+depends="desktop-file-utils"
+short_desc="Qt4 GUI program used to map keyboard keys and mouse controls to a gamepad"
```

Comment:
But now `short_desc` is too long!
````
$ xlint srcpkgs/antimicro-qt4/template
srcpkgs/antimicro-qt4/template:11: short_desc should be less than 72 chars
````

Reply:
Change to something like:
````
short_desc="Qt4 GUI program used to map keyboard and mouse controls to a gamepad"
````

Reply:
Or maybe "Map keyboard and mouse controls to a gamepad"